Makefile

```diff
@@ -0,0 +1,52 @@
+.PHONY: all docker docker-dev docker-tools dev dep build test mock protobuf changelog clean 
+
+all: clean test build docker
+
+MAJOR_VERSION := $(shell git describe --abbrev=0 | cut -d . -f 1)
+MINOR_VERSION := $(shell git describe --abbrev=0 | cut -d . -f 1-2)
+PATCH_VERSION := $(shell git describe --abbrev=0)
+DEV_VERSION := $(shell git describe)
```

Comment:
I have a version 0.14.2 with git describe where I actually have a v0.15 in my tags.
Is that something normal? I'm afraid that the CI will generate the wrong version

Reply:
Following https://git-scm.com/docs/git-describe

> The command finds the most recent tag that is reachable from a commit. 

So probably, your tag v0.15 is on a different branch and it's not reachable from the building commit. Otherwise, `git describe` shows the most recent tag.

Reply:
Even from `dev` and `master` my `git describe` shows `v0.14.2-152-gd3f71c74` but we did release v0.15. Can you try and tell me what you've got

Reply:
Yes, on `dev` branch the latest tag is `v0.14.2` because tag v0.15.0 is only on master branch.

The commit merge `9062446d7cbc049c8482f0606703e3154331a017` has that tag, and that one is only on master branch. So it is working as expected and the master release will work. 

We should use tag only in one way to have it clear:

1) use on dev branch (will be merged into master with all tags)
2) use only on master branch

For now we tag sometimes on dev, sometimes on master.

Reply:
I think this is source of problems for now and we don't use the `docker` command anyway. What do you think about just removing/commenting this for now. I'm afraid we will want to use it for the CI and this will mess up everything.

Reply:
> I think this is source of problems 

What pbms do you mean? The makefile is just to use locally (you can also build a docker image to check if everything will work on ci). The real source of the problem is the inconsistent tagging policy.

> don't use the docker command anyway.

Yep, it's there only for us to testing 

> What do you think about just removing/commenting this for now.

If you want so, I can remove, comment it, but as I said previously. I intentionally didn't use that makefile anywhere - it's for local dev only.

Reply:
We might be tempted to change the CI to use the makefile instead, to have all our scripts in one and if we have the wrong versioning that will be bad